lib/engine/step/buy_sell_par_shares.rb

```diff
@@ -159,8 +159,10 @@ def process_par(action)
       def pass!
         super
         if @current_actions.any?
+          @round.pass_order.delete(current_entity)
           current_entity.unpass!
         else
+          @round.pass_order << current_entity unless @round.pass_order.include?(current_entity)
```

Comment:
@round.pass_order |= [current_entity]

Reply:
|= appends to the existing array?

Reply:
Basically, append current_entity to the array if it isn't already present in the array.

Reply:
yes, it's a set union operator

Reply:
Nice. Learn something new every day.